game-engine/src/main/scala/za/co/entelect/challenge/game/contracts/game/CarReferee.scala

```diff
@@ -1,13 +1,24 @@
 package za.co.entelect.challenge.game.contracts.game
 
+import za.co.entelect.challenge.game.contracts.Config.Config
 import za.co.entelect.challenge.game.contracts.common.RefereeMessage
 import za.co.entelect.challenge.game.contracts.map.GameMap
 
 import scala.collection.JavaConverters._
 
 class CarReferee extends GameReferee {
   override def isMatchValid(gameMap: GameMap): RefereeMessage = {
-    //TODO: determine what would invalidate a match
+    val maxRoundsExceeded = gameMap.getCurrentRound() > Config.MAX_ROUNDS;
+    if(maxRoundsExceeded)
```

Comment:
One day we can add checks for excessive scores, speed, powerups count and such here

Reply:
Speed would crash the game, fun fact, can probably do score and powerups count. Let me give it a bash

Reply:
Sorted. 👍